tests/icelake_config_id/icelake_config_id.edl

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave {
+    from "openenclave/edl/syscall.edl" import *;
```

Comment:
Instead of importing everything, please refer to https://github.com/openenclave/openenclave/blob/8f6691ef4927477f17ea7f338c57ab561fb7ffe5/tests/echo/echo.edl#L5 to import minimum list of ocall/ecalls, which supports `printf` or `oe_host_printf` used by tests.